src/components/chip/chip.e2e.ts

```diff
@@ -173,4 +183,60 @@ describe("calcite-chip", () => {
   });
 
   it("supports translation", () => t9n("calcite-chip"));
+  /*
```

Comment:
Can we remove this if its not going to be used?

Reply:
Yep working on adding these in, will remove any comments.